docs/vexim-acl-check-rcpt.conf

```diff
@@ -52,5 +52,16 @@
 #                          and spamassassin='1'}}}{1} {yes}{no}}
 #          !acl          = spf_rcpt_acl
 
+#
+# To use a DNSBL the best would be to set up a caching dns server for you own
+# otherwise you might get rate limited and your requests are not processed
+# 
+# Also disable spamchecking for authenticated users because dial up networks
+# are often marked as bad in case of sending mails
+#
+  accept
+          authenticated = *
+          condition = ${if eq{$authenticated_id}{$sender_address}{yes}{no}}
```

Comment:
If I understand this correctly, it depends on whether or not we want to keep checking authenticated users. `accept authenticated=*` will mean that this ACL has passed with an accept, and no further checks (DNSBL or any other) will be performed. Merging that condition into the DNSBL block will mean that either the user will be denied (if not authenticated and in DNSBL), or checks will proceed further. Am I correct?